builtin/logical/pki/path_config_crl.go

```diff
@@ -85,9 +86,99 @@ the NextUpdate field); defaults to 12 hours`,
 		Operations: map[logical.Operation]framework.OperationHandler{
 			logical.ReadOperation: &framework.PathOperation{
 				Callback: b.pathCRLRead,
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: "OK",
+						Fields: map[string]*framework.FieldSchema{
+							"expiry": {
+								Type: framework.TypeString,
+								Description: `The amount of time the generated CRL should be
+valid; defaults to 72 hours`,
+								Default: "72h",
```

Comment:
I don't know if these defaults are meaningful for response fields since that's the actual field we are requesting. I'd be tempted to  skip the `Default` field, but I guess it's OK to leave it as is.

Reply:
Yeah. I also figured it didn't need to be there... I can remove them.

Reply:
Removed.